src/components/ContactForm/ContactForm.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import ContactForm from "./ContactForm";
+
+export default {
+  title: "Components/ContactForm",
+  component: ContactForm,
+} as Meta;
+
+const Template: StoryFn<{ isOpen: boolean }> = ({ isOpen }) => {
+  const [modalOpen, setModalOpen] = useState(isOpen);
+
+  return (
+    <>
+      <button onClick={() => setModalOpen(true)}>Open Contact Form</button>
```

Comment:
Not a necessary change, but one that I believe would enhance the storybooks user experience, would be to add some styling to this Open contact form button.
<img width="780" alt="image" src="https://github.com/user-attachments/assets/db12bb57-0857-4200-a55f-1d9c21d6e2e2"> 
Here is what it looks like from the Modal stories tsx file 
<img width="780" alt="image" src="https://github.com/user-attachments/assets/cfb25a93-9b29-47eb-9c5e-2ad3fd2b5837">
Which you can just take this code here to add some styling to this component:
        className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full"
 This will make it match with the modal stories tsx button. (Attached in the 2nd screenshot)